test/jdk/com/sun/tools/attach/warnings/Application.java

```diff
@@ -39,6 +47,10 @@ public static void main(String[] args) throws Exception {
 
             // send pid
             long pid = ProcessHandle.current().pid();
+            while (!AttachHandler.isAttachApiInitialized()) {
+                // delay 2s to allow AttachAPI initialization
+                Thread.currentThread().sleep(2000);
```

Comment:
2 sec is a long time, I'm guessing a shorter time like 100ms would be sufficient. The tests will run faster, in particular if we need this same change elsewhere.

Reply:
Changed to `100ms`